patches/0003-patches-openwrt-add-TL-WR841ND-N-8M-and-16M-variants.patch

```diff
@@ -1,43 +1,42 @@
-From 0db9016d1646448272d4fafe43b720d10416ddbd Mon Sep 17 00:00:00 2001
+From 58fb8a2888b9096d9000326c7d239e70166469d7 Mon Sep 17 00:00:00 2001
 From: Dark4MD <github.web@manu.li>
-Date: Thu, 5 May 2022 18:04:46 +0200
-Subject: [PATCH] patches/openwrt: add TL-WR841ND/N 8M and 16M variants to
- ath79
+Date: Sun, 6 Nov 2022 18:59:21 +0100
+Subject: [PATCH] ath79-generic: add TL-WR841ND/N 8M and 16M variants
 
 ---
- ...841ND-N-8M-and-16M-variants-to-ath79.patch | 753 ++++++++++++++++++
+ ...add-TL-WR841ND-N-8M-and-16M-Variants.patch | 843 ++++++++++++++++++
  targets/ath79-generic                         |  40 +
- 2 files changed, 793 insertions(+)
- create mode 100644 patches/openwrt/0009-add-TL-WR841ND-N-8M-and-16M-variants-to-ath79.patch
+ 2 files changed, 883 insertions(+)
+ create mode 100644 patches/openwrt/0010-add-TL-WR841ND-N-8M-and-16M-Variants.patch
 
-diff --git a/patches/openwrt/0009-add-TL-WR841ND-N-8M-and-16M-variants-to-ath79.patch b/patches/openwrt/0009-add-TL-WR841ND-N-8M-and-16M-variants-to-ath79.patch
+diff --git a/patches/openwrt/0010-add-TL-WR841ND-N-8M-and-16M-Variants.patch b/patches/openwrt/0010-add-TL-WR841ND-N-8M-and-16M-Variants.patch
 new file mode 100644
-index 00000000..d803a772
+index 00000000..3c054260
 --- /dev/null
-+++ b/patches/openwrt/0009-add-TL-WR841ND-N-8M-and-16M-variants-to-ath79.patch
-@@ -0,0 +1,753 @@
-+From f1be647c25c3b1c393add1cdba6b2ad2d23e0b71 Mon Sep 17 00:00:00 2001
++++ b/patches/openwrt/0010-add-TL-WR841ND-N-8M-and-16M-Variants.patch
```

Comment:
I liked the lowercase commit message more; any reason to revert that?
Secondly the new patchnumber should become 0011.

```suggestion
+++ b/patches/openwrt/0011-add-TL-WR841ND-N-8M-and-16M-variants-to-ath79.patch
```